src/librustc_interface/util.rs

```diff
@@ -272,7 +270,7 @@ pub fn get_codegen_backend(sess: &Session) -> Box<dyn CodegenBackend> {
             filename if filename.contains(".") => {
                 load_backend_from_dylib(filename.as_ref())
             }
```

Comment:
Maybe move this to `get_builtin_codegen_backend` and rename that function to `get_codegen_backend`, while renaming this function to `get_codegen_backend_for_sess` or even inline this function.

Reply:
I'm not sure I understand? What would the purpose of that movement be?

Reply:
The purpose of that change is that the version which doesn't take a `Session` as input will accept `/path/to/backend.so`. I am fine with keeping it the way it is right now.